Vagrantfile

```diff
@@ -0,0 +1,44 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  
+  #redirect traffic from port 5000 on the host machine to port 5000 on the VM
+  config.vm.network "forwarded_port", guest: 5000, host: 5000  
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+  sudo apt-get update
+  sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+  libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+  xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+  
+  #Install pyenv after clearing out ~/.pyenv
+  #rm -r ~/.pyenv
```

Comment:
If you want this line so that you clear out the pyenv directory when you reprovision the VM, don't comment it out.

If you've decided it's not useful, delete it and update the comment on line 12.

Or if you want to keep it because it's sometimes useful, keep it like this but update the comment on line 12 to say "uncomment the following line to reprovision with a fresh pyenv installation"